foundations/src/telemetry/settings/tracing.rs

```diff
@@ -26,6 +26,13 @@ pub struct TracingSettings {
 
     /// The strategy used to sample traces.
     pub sampling_strategy: SamplingStrategy,
+    /// Enable liveness tracking of all generated spans. Even if the spans are
+    /// unsampled. This can be useful for debugging potential hangs cause by
+    /// some objects remaining in memory.  The default value is false, meaning
+    /// only sampled spans are tracked.
+    ///
+    /// To get a json dump of the currently active spans, query: `/debug/traces`
```

Comment:
Nit: worth mentioning here that we are talking about querying telemetry server's route here. It's confusing out of the context